crates/rspack_plugin_lightning_css_minimizer/src/lib.rs

```diff
@@ -233,7 +234,7 @@ async fn process_assets(&self, compilation: &mut Compilation) -> Result<()> {
             .map_err(|e| error!(e.to_string()))?;
           let result = stylesheet
             .to_css(PrinterOptions {
-              minify: true,
+              minify: minimizer_options.minify.unwrap_or(false),
```

Comment:
Why change this option to default `false`?

Reply:
A minimizer should minify code by default

Reply:
I think we do not need to add a `minify` option to the minimizer plugin, it does not make sense

Reply:
Thanks. I have checked out